README.md

```diff
@@ -0,0 +1,52 @@
+# Difflint #
+
+Do you have a codebase with inconsistent code style, and you want to
+start checking your commits with a linter; but you don't want to rewrite
+the entire repository, since it will mess up your history and the
+ability to `git bisect`?
+
+**Difflint** will check only the changes that you make in each commit.
+It's suitable for using as a Git pre-commit hook.
+
+## Linters supported ##
+
+- [JSCS](http://jscs.info) (JavaScript)
+- [JSHint](http://jshint.com/docs) (JavaScript)
+
+## Requirements ##
+
+- Python 3.3 or later
+- JSCS
+- JSHint
+- NPM for installing JSCS and JSHint
+
+## Installation ##
+
+- Clone this repository, and run `python3 setup.py install`.
```

Comment:
I wonder if we can bring down the number of steps in the installation process in the future. I have a feeling that will be the main barrier to adoption for this project. Maybe the install script could have an option where it configures a git template with the pre-commit hook (for future git repos) and/or recursively traverses the file system, installing the pre-commit in .git/hooks for any existing git repos. In the meantime, this documentation seems very clear to me.